core/src/test/scala/org/apache/spark/scheduler/TaskSchedulerImplSuite.scala

```diff
@@ -139,29 +139,6 @@ class TaskSchedulerImplSuite extends SparkFunSuite with LocalSparkContext with B
     assert(!failedTaskSet)
   }
 
-  test("Scheduler does not crash when tasks are not serializable") {
```

Comment:
If I understand right, you are saying there are now test cases which cover all of the little individual pieces inside a `Task`, so we don't need an explicit test for the serializing the entire task.